src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -454,4 +466,61 @@ public VisualEditorEditTemplateDialog openEditTemplateDialog() {
     clickContextMenu();
     return new VisualEditorEditTemplateDialog(driver);
   }
+
+  public VisualEditorPageObject clickInsertToolButton() {
+    for (int i=0; i<toolsList.size(); i++)
+    {
+      if (toolsList.get(i).getText().contentEquals("Insert"))
+      {
+        toolsList.get(i).click();
+      }
+    }
+    return this;
+  }
+
+  public VisualEditorPageObject clickInsertInfoboxFromInsertToolMenu() {
+    for (int i=0; i<insertMenuTools.size(); i++)
+    {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this left curly brace to the end of previous line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)